Python/Tests/Core.UI/ErrorListTaskListTests.cs

```diff
@@ -220,5 +220,18 @@ internal struct TaskItemInfo {
             app.WaitForTaskListItems(typeof(SVsErrorList), 0);
             app.WaitForTaskListItems(typeof(SVsTaskList), 0);
         }
+
+        /// <summary>
+        /// Make sure *.pyi files ignore the active Python version
+        /// </summary>
+        public void ErrorListEmptyForValidTypingFile(VisualStudioApp app) {
+            var project = app.OpenProject(@"TestData\Typings.sln");
+            project.ProjectItems.Item("mymod.pyi").Open();
```

Comment:
how does it find the error in usermod.py if the only file opened is mymod.pyi? Do you need to also open usermod.py?

Reply:
All `<Compile>` files in a project contribute to the error list, but right now stub files only do when they are open (they default to `<Content>` or `<None>`, IIRC).